### PR TITLE
fixed typos in backup.rst file

### PR DESCRIPTION
**Pull request by fakela**
#### Summary

fixed typos in backup.rst file

### CODE REVIEW

File: source/administration/backup.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Options to protect your Mattermost server from different types of failures range
 Backup
 ------
 
-The state of your Mattermost server is contained in multiple data stores that need to be separately backed-up and restored to fully recover your system from a failure. 
+The state of your Mattermost server is contained in multiple data stores that need to be separately backed-up and restored to fully recover your system from failure.
```
Comment (justinegeffen) [inline source/administration/backup.rst:9]:
```suggestion
The state of your Mattermost server is contained in multiple data stores that need to be separately backed up and restored to fully recover your system from failure. 
```

**Comment by justinegeffen** [comment]
Thanks, @fakela, this is a great catch! I have added one suggestion - let me know what you think and then I think this is good to merge.

**Comment by fakela** [comment]
@justinegeffen thanks it's looks good I have committed the suggestions

File: source/administration/backup.rst [hunk 2]
```diff
@@ -38,19 +38,19 @@ There are two common approaches:
 
 Automated backup
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Automating backup up a Mattermost server provides a copy of the server's state at a particular point in time, which can be restored if a failure in future leads to loss of data. Options include: 
+Automating backup up a Mattermost server provides a copy of the server's state at a particular point in time, which can be restored if a failure in the future leads to loss of data. Options include:
```
Comment (justinegeffen) [inline source/administration/backup.rst:41]:
```suggestion
Automating backup for a Mattermost server provides a copy of the server's state at a particular point in time, which can be restored if a failure in the future leads to loss of data. Options include: 
```

File: source/administration/backup.rst [hunk 3]
```diff
@@ -38,19 +38,19 @@ There are two common approaches:
 
 Automated backup
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Automating backup up a Mattermost server provides a copy of the server's state at a particular point in time, which can be restored if a failure in future leads to loss of data. Options include: 
+Automating backup up a Mattermost server provides a copy of the server's state at a particular point in time, which can be restored if a failure in the future leads to loss of data. Options include: 
 
 - Automation to periodically backup the Mattermost server, which may include all the components listed above or a subset depending on what you choose to protect.
```
Comment (justinegeffen) [inline source/administration/backup.rst:43]:
```suggestion
- Automation to periodically back up the Mattermost server, which may include all the components listed above or a subset depending on what you choose to protect.
```

**Comment by justinegeffen** [comment]
I picked up two minor edits which I've committed and this looks good to go. Thanks, @fakela! :)

**Pull request merged**